src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -112,6 +112,8 @@
         'DateTimeZone' => array('Symfony\Component\VarDumper\Caster\DateCaster', 'castTimeZone'),
         'DatePeriod' => array('Symfony\Component\VarDumper\Caster\DateCaster', 'castPeriod'),
 
+        'GMP' => array('Symfony\Component\VarDumper\Caster\GmpCaster', 'castInt'),
```

Comment:
not all GMP object's values can be represented as strings, so castString should be the implementation